package.json

```
@@ -28,14 +28,14 @@
     "build": "tsc -p ."
   },
   "dependencies": {
-    "chalk": "^5.0.1",
-    "log-update": "^5.0.1",
+    "chalk": "^5.4.1",
+    "log-update": "^6.1.0",
     "sisteransi": "^1.0.5"
   },
   "devDependencies": {
-    "@changesets/cli": "^2.24.4",
-    "@types/node": "^18.7.16",
-    "typescript": "^5.2.2"
+    "@changesets/cli": "^2.27.12",
+    "@types/node": "^22.10.10",
```

Astro still supports Node 18, so we should probably stick to that version of the types to avoid using any APIs introduced only in Node 20/22:

```suggestion
    "@types/node": "^18.7.16",
```

---

Ah yeah fair point, reverted